app/src/main/java/uk/gov/android/securestore/crypto/RsaCryptoManager.kt

```diff
@@ -1,22 +1,38 @@
 package uk.gov.android.securestore.crypto
 
+import android.os.Build
 import android.security.keystore.KeyGenParameterSpec
 import android.security.keystore.KeyProperties
 import android.util.Base64
+import androidx.annotation.RequiresApi
+import androidx.fragment.app.FragmentActivity
+import java.security.GeneralSecurityException
 import java.security.KeyPairGenerator
 import java.security.KeyStore
 import java.security.KeyStore.PrivateKeyEntry
 import javax.crypto.Cipher
+import uk.gov.android.securestore.AccessControlLevel
+import uk.gov.android.securestore.authentication.Authenticator
+import uk.gov.android.securestore.authentication.AuthenticatorCallbackHandler
+import uk.gov.android.securestore.authentication.AuthenticatorPromptConfiguration
+import uk.gov.android.securestore.authentication.UserAuthenticator
 
 /**
  * Implementation of [CryptoManager] using RSA encryption algorithm to create Public/Private key pair.
  */
-internal class RsaCryptoManager : CryptoManager {
+internal class RsaCryptoManager(
+    private val context: FragmentActivity,
+    private val alias: String,
+    private val accessControlLevel: AccessControlLevel,
+    private val authenticator: Authenticator = UserAuthenticator(context)
```

Comment:
We only seem to be using the context to create the default value of the `authenticator`. We able to just remove the `context` constructor parameter and just give it an authenticator instead?